packages/contracts/test/registry/plugin-repo-registry.ts

```diff
@@ -157,4 +157,38 @@ describe('PluginRepoRegistry', function () {
       .to.be.revertedWithCustomError(ensSubdomainRegistrar, 'AlreadyRegistered')
       .withArgs(pluginRepoNameDomainHash, ensSubdomainRegistrar.address);
   });
+
+  it('should revert if name contains any invalid char', async () => {
+    const baseName = 'this-is-my-super-valid-name';
+
+    // loop through the ascii table
+    for (let i = 0; i < 127; i++) {
+      // deploy a pluginRepo and initialize
+      const newPluginRepo = await deployNewPluginRepo(ownerAddress);
+
+      // random place for the char to improve validation check
+      const placement = Math.round(Math.random() * baseName.length);
```

Comment:
Random testing could wrongly pass on a change we make, although it would have failed with a different seed / entropy.
We should at least print the RNG seed to be able to reproduce the failing tests.
`Math.random` apparently doesn't allow to print the seed.

Probably fine for now to leave it as it is due to the lack of time, but this should be changed in the future.

Reply:
Agreed I added another test to check for strings longer than 32 bytes which should solve this